lib/service/service.go

```diff
@@ -4082,7 +4082,16 @@ func (process *TeleportProcess) setupProxyTLSConfig(conn *Connector, tsrv revers
 		if acmeCfg.URI != "" {
 			m.Client = &acme.Client{DirectoryURL: acmeCfg.URI}
 		}
-		tlsConfig = m.TLSConfig()
+		// We have to duplicate the behavior of `m.TLSConfig()` here because
+		// http/1.1 needs to take precedence over h2 due to
+		// https://bugs.chromium.org/p/chromium/issues/detail?id=1379017#c5 in Chrome.
+		tlsConfig = &tls.Config{
+			GetCertificate: m.GetCertificate,
+			NextProtos: []string{
+				"http/1.1", "h2", // enable HTTP/2
+				acme.ALPNProto, // enable tls-alpn ACME challenges
```

Comment:
acme is appended on line 4097

Reply:
Huh. That append on line 4097 doesn't actually make much sense given the previous use of `m.TLSConfig()`. That function returns:

```
	return &tls.Config{
		GetCertificate: m.GetCertificate,
		NextProtos: []string{
			"h2", "http/1.1", // enable HTTP/2
			acme.ALPNProto, // enable tls-alpn ACME challenges
		},
	}
```

Which means the later append of `acme.ALPNProto` was effectively a noop. I'm going to remove the later append here and then adjust the constants.